index.bs

```diff
@@ -2337,9 +2337,9 @@ Although operations {{MLGraphBuilder/greaterOrEqual()}} and {{MLGraphBuilder/les
         1. If that returns failure, then [=exception/throw=] a {{TypeError}}.
     1. *Make graph connections:*
         1. Let |output| be the result of [=creating an MLOperand=] given [=this=] and |descriptor|.
-        1. Let |operator| be an [=operator=] for the binary operation |op|, given |a| and |b|.
+        1. Let |operator| be an [=operator=] for the logical operation |op|, given |a| and (if |op| is not "not") |b|.
         1. Set |output|.{{MLOperand/[[operator]]}} to |operator|.
-        1. Set |operator|'s [=operator/inputs=] to |a| and |b|.
+        1. Set |operator|'s [=operator/inputs=] to |a| and (if |op| is not "not") |b|.
```

Comment:
How do you feel about pulling `logicalNot` (and likely a future `bitwiseNot`) out into a separate category of "element-wise-unary-logical-op" and making this one "element-wise-binary-logical-op"?

Reply:
I'd say "yes but later" i.e. get the ugly fix in, refactor later. I've thought the carve-outs for "not" were a little ugly.

Also, I can reword to eliminate "not not" either trivially eg. `is anything other than "not"` or by introducing a top level "if" step rather than a nested parenthetical expression.

Reply:
👍

Reply:
> I can reword to eliminate "not not" either trivially

I like it.

Reply:
done!

Reply:
p.s. love the commit message.